<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*
	color: rgb(85, 118, 189);
	blue-primary: rgb(85,118,189);
	blue-active: rgb(68,94,151);
	border-frame:  rgb(220, 220, 220);
	primary-background: rgb(240, 240, 245);
	icon-color: rgb(30,40,70);
	btn_pressed-bg: background-color: rgb(241,243,246);

	subtitle: color: #888;

	orange: #ffa428;
*/

*{
	font: 14px &quot;Microsoft YaHei&quot;;
	color: rgb(30,40,70);
	background-color: rgb(240, 240, 245);
	outline: none;
	border:none;
}

/* ToolTip */
QToolTip {
	color: rgb(30,40,70);
    border: 1px solid rgba(180, 180, 180, 100);
    padding: 5px;
    border-radius: 4px;
    opacity: 150;
}

/* Menu */ 
QMenu{
	background-color: #fff;
	padding-top: 8px;
	padding-bottom: 8px;
}
QMenu::item{
	padding: 5px 10px 5px 25px;
	border-left: 10px solid transparent;
}
QMenu::item:selected{
	color: #fff;
	background-color: #5576be;
}
QMenu::item:pressed{
	background-color: #445e97;
}
QMenu::separator{
    height: 1px;
    background: #eee;
    margin-left: 10px;
    margin-right: 10px;
}
QMenu:disabled{
	color:  rgba(50,50,80,80);
}

/* Widget */
#navMenuWidget , #toolBarWidget, #headerWidget
{
	background: #fff;
}

/* ProgressBar */
QProgressBar
{
    color: transparent;
}
QProgressBar::chunk {
    background-color: #ffa428;
}
#pbRating_lib
{
    max-height: 4px;
}


/* Label */
QLabel
{
	background:transparent;
}
#lbHeaderName
{
	font-size: 20px;
	font-weight: bold;
	padding-left: 10px;
}
#lbCategoryName
{
	min-height: 20px;
	font-size: 20px;
	font-weight: bold;
}
#lbBookCount
{
	min-height: 16px;
	font-size: 16px;
	color: rgb(180,180,180);
}
#lbNickName
{
	font-weight: bold;
}
#lbUserNickName
{
	font-size: 20px;
	font-weight: bold;
	padding-left: 10px;
}

/**************************************************/
/* 					Library 				      */
/**************************************************/

/* Label */
#lwLentBooksView,
#lbBookTitle_lib
{
	font-size: 18px;
	font-weight: bold;
}
#lbAuPubTime_lib,
#lbSummary_lib
{
	color: #888;
}
#lbRating_lib
{
	color: #ffa428
}

/* QListView */
QListView::item
{
	background-color: rgb(255, 255, 255);
	padding: 10px;
	border-radius: 4px;
	border: 0 solid transparent;
}
QListView::item:hover {
	padding: 8px;
    border: 2px solid rgb(85,118,189);
}
QListView::item:selected {
	padding: 8px;
    border: 2px solid rgb(187,200,228);
}
QListView::item:selected:!active {
	padding: 8px;
    border: 2px solid #fff;
}
QListView::item:selected:active {
	padding: 8px;
    border: 2px solid rgb(187,200,228);
}
QListView::item:hover {
	padding: 8px;
    border: 2px solid rgb(224,227,241);
}
#lwLentBooksView::item
{
	color: rgb(30,40,70);
	max-width: 130px;
	max-height: 220px;
}
#lwLibraryView::item
{
	width: 480px;
	height: 180px;
}

/* LineEdit */

QLineEdit
{
	height: 40px;
	padding: 0px 20px;
	border-radius: 20px;
	background: white;
    /*border: 1px solid rgb(220, 220, 220);*/
}
/*QLineEdit::hover
{
	border: 1px solid rgb(85,118,189);;
}*/

/* PushButton */

#btnLoadMore,
#btnLogout
{
	color: white;
	min-height: 30px;
	border-radius: 4px;
	background-color: rgb(85,118,189);
}
#btnLoadMore:hover,
#btnLogout:hover
{
	background-color: rgb(68,94,151);
}

/* Navgation Bar Menus */

#btnBookShelf,
#btnBookGarden,
#btnSetting
{
	height: 50px;
	border-radius: 0;
	background-color: transparent;
}
#btnBookShelf:hover,
#btnBookGarden:hover,
#btnSetting:hover
{
	padding-left: -4px;
	border-left: 4px solid rgb(85,118,189);
	background-color: rgb(232,240,255);
}
#btnBookShelf:checked,
#btnBookGarden:checked,
#btnSetting:checked
{
	padding-left: -4px;
	border-left: 4px solid rgb(85,118,189);
	background-color: rgb(210,226,255);
}


/* ScrollBar  */

QScrollBar:vertical
{
    background: white;
    width: 10px;
}
QScrollBar::handle:vertical
{
    background: rgb(224,227,241);
    min-height: 10px;
}
QScrollBar::handle:hover
{
	background: rgb(187,200,228);
}
QScrollBar::add-line:vertical
{
    height: 0;
}
QScrollBar::sub-line:vertical
{
    height: 0;
}


QScrollBar:horizontal
{
    background: white;
    height: 10px;
}
QScrollBar::handle:horizontal
{
    background: rgb(224,227,241);
    min-width: 10px;
}
QScrollBar::horizontal:hover
{
	background: rgb(187,200,228);
}
QScrollBar::add-line:horizontal
{
    width: 0;
}
QScrollBar::sub-line:horizontal
{
    width: 0;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" rowspan="2">
    <widget class="QWidget" name="navMenuWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="topMargin">
         <number>15</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <property name="verticalSpacing">
         <number>10</number>
        </property>
        <item row="1" column="0" colspan="3">
         <widget class="QLabel" name="lbNickName">
          <property name="text">
           <string>点击头像登录</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="AvatorWidget" name="lbAvator">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnBookShelf">
        <property name="text">
         <string>我的书架</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/Icons/DefaultIcons/book_shelf.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">navButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBookGarden">
        <property name="text">
         <string>图书馆</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/Icons/DefaultIcons/book_garden.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">navButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSetting">
        <property name="text">
         <string>应用设置</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/Icons/DefaultIcons/setting.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">navButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageBookShelf">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="BookViewWidget" name="lwLentBooksView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="iconSize">
          <size>
           <width>130</width>
           <height>180</height>
          </size>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="movement">
          <enum>QListView::Static</enum>
         </property>
         <property name="resizeMode">
          <enum>QListView::Adjust</enum>
         </property>
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="viewMode">
          <enum>QListView::IconMode</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageBookGarden">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnLoadMore">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>加载更多</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="BookViewWidget" name="lwLibraryView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="movement">
          <enum>QListView::Static</enum>
         </property>
         <property name="flow">
          <enum>QListView::LeftToRight</enum>
         </property>
         <property name="resizeMode">
          <enum>QListView::Adjust</enum>
         </property>
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="viewMode">
          <enum>QListView::IconMode</enum>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>lwLibraryView</zorder>
      <zorder>btnLoadMore</zorder>
     </widget>
     <widget class="QWidget" name="pageSetting">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="lineEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string>CheckBox</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QWidget" name="headerWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="lbHeaderName">
          <property name="text">
           <string>MonoKelvin</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="leSearchBox">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>搜索...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QWidget" name="toolBarWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>headerWidget</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>navMenuWidget</zorder>
  <zorder>toolBarWidget</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AvatorWidget</class>
   <extends>QLabel</extends>
   <header location="global">Widgets/AvatorWidget.h</header>
  </customwidget>
  <customwidget>
   <class>BookViewWidget</class>
   <extends>QListWidget</extends>
   <header location="global">Widgets/BookViewWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="navButtonGroup"/>
 </buttongroups>
</ui>
